ets-archetypes-application/src/main/resources/archetype-resources/src/main/scala/Main.scala

```diff
@@ -0,0 +1,5 @@
+package ${package}
+
+object Main extends App {
+  println(s"This is the main class for the ${artifactId} application")
+}
```

Comment:
Nitpicking: can you please add a newline at the end of the file?

Reply:
Fixed.